includes/repositories/EUtilsBioSampleRepository.php

```diff
@@ -37,14 +37,30 @@ public function create($data) {
     // Throw an exception if a required field is missing
     $this->validateFields($data);
 
-    // Create the base record
-    $description = is_array($data['description']) ? implode("\n",
-      $data['description']) : $data['description'];
+    if (!$data['organism']) {
+      throw new Exception(
+        'Unable to create BioSample due to lack of organism information'
+      );
+    }
+
+    // getOrganism throws an exception so without try/catch, we will
+    // automatically fail and exit
+    $organism = $this->getOrganism($data['organism']['taxonomy_id']);
+    $organism = $organism->organism_id;
```

Comment:
i always find it confusing when we pass around record_ids vs the whole records.  Note the `createBioSample` call below, it has `$contact->contact_id` AND `$organism`. 

IMO we should always specify that its an id if its just the ID, otherwise assume its teh whole record.

```suggestion
```